jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/ssl/SslContextFactory.java

```diff
@@ -95,7 +96,7 @@
  * and {@link Client} to configure HTTP or WebSocket clients.</p>
  */
 @ManagedObject
-public abstract class SslContextFactory extends AbstractLifeCycle implements Dumpable
+public abstract class SslContextFactory extends ContainerLifeCycle implements Dumpable
```

Comment:
Seems like a big change.  AbstractLifeCycle to ContainerLifeCycle

Reply:
Is it? @gregw made the exact same change in the original redux 3 PR so I thought this was okay.

Reply:
Ah, he did? oh well, disregard my earlier comment.

Reply:
Perhaps `SslContextFactory` should only take `Path` and/or `Resource` as configurations, not `String`, then we can skip the whole `ResourceFactory` / Mount management effort within this class.

Let the users of the class manage it.

The only impact I see is how to make that work from a jetty XML configuration.